.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,54 @@
+name: phpspec
+
+on:
+  pull_request:
```

Comment:
should also run on pushes (at least on the main branch and tags)

Reply:
Done